packages/ui/src/Input/index.tsx

```diff
@@ -0,0 +1,30 @@
+import { HTMLAttributes } from "react";
+
+interface IInputProperties extends HTMLAttributes<HTMLInputElement> {
+  label?: string;
+  name?: string;
+  error?: string;
```

Comment:
lets make error a boolean and add errorMessage for message, there could be use case where we want to indicate error without a message.